### PR TITLE
Eng 1507 add stack trace to failed object deletion in workflow deletion [UI/3]

### PR DESCRIPTION
**Pull request by eunice-chan**
## Describe your changes and why you are making these changes
Add stack traces / error messages to the failed object deletions as suggested in https://github.com/aqueducthq/aqueduct/pull/290.

BE Component https://github.com/aqueducthq/aqueduct/pull/340
UI Component https://github.com/aqueducthq/aqueduct/pull/341
SDK Component https://github.com/aqueducthq/aqueduct/pull/342

## Demo
Shows error with failed deletion:
image

0:17 - Deletes 5 tables and shows the error along with failed deletion. Scrolls because there are too many results on the screen.

https://user-images.githubusercontent.com/30596854/184263334-07a1d32d-e9a9-489b-9b92-caba2044f804.mov

## Related issue number (if any)
ENG 1370
ENG 1507

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/workflows/WorkflowSettings.tsx [hunk 1]
```diff
@@ -629,33 +635,44 @@ const WorkflowSettings: React.FC<WorkflowSettingsProps> = ({
...
+                    <ListItemText
+                      primary={displayObject(
+                        integrationName,
+                        objectResult.name
+                      )}
+                    />
+                  </ListItem>
+                  {objectResult.exec_state.status ===
+                    ExecutionStatus.Failed && (
+                    <Alert severity="error">
```
Comment (vsreekanti) [inline src/ui/common/src/components/workflows/WorkflowSettings.tsx:671]: Let's also use an `AlertTitle` inside this `Alert`. It'll make the formatting a little easier to read.

**Review by vsreekanti — APPROVED** [review]
@eunice-chan, mostly looks good! A couple things: 
* I left a comment inline about adding a title to the alert.
* In the screenshot, it says 4 tables successfully deleted, but it only shows one name with a green checkmark next to it. Let's try to make that consistent. 
* Can we make this modal `fullWidth`? I think it'll help the formatting of the error message. I think there's some leading whitespace as well that we can trim.

**Comment by eunice-chan** [comment]
@vsreekanti
> In the screenshot, it says 4 tables successfully deleted, but it only shows one name with a green checkmark next to it. Let's try to make that consistent.

That's because there is a scrollbar and if you scroll down, there are 3 other tables. But that is a good point that the user may not know the field is scrollable. What would be the best way to make that obvious?

**Comment by vsreekanti** [comment]
I would make the div adjust to the height of the content so there's no scroll (but maybe set a max height for really long lists?). @agiron123 might know better, but you could try setting the height of the box to be `min-content`.

**Review by agiron123 — APPROVED** [review]

**Comment by eunice-chan** [comment]
Here is the updated look!

Screencap from the video:
The whole modal is scrollable instead of just the list of saved objects deletion results.

image

Video:

https://user-images.githubusercontent.com/30596854/184448666-d7c6a208-fbfd-4c5c-9f5e-c9ad10b766e7.mov

**Comment by vsreekanti** [comment]
This looks great, thanks Eunice!

**Pull request merged**